app/views/components/pending_daily_statement_v2.scala.html

```diff
@@ -0,0 +1,100 @@
+@*
```

Comment:
Can we check over the indentation of this file? it looks a little off to me

Reply:
This file is not needed now. It has been deleted.